ui/src/app/app-routing.tsx

```diff
@@ -61,21 +70,41 @@ export const AppRoutingComponent: React.FunctionComponent<RoutingProps> = ({onSi
     />
 
     <ProtectedRoutes guards={[signInGuard]}>
-        <AppRoute
+      <AppRoute
+        path='/'
+          component={() => <HomepagePage routeData={{title: 'Homepage'}}/>}
+      />
+      <AppRoute
+          path='/admin/user-audit'
+          component={() => <UserAuditPage routeData={{title: 'User Audit'}}/>}
+      />
+      <AppRoute
+          path='/admin/user-audit/:username'
+          component={() => <UserAuditPage routeData={{title: 'User Audit'}}/>}
+      />
+      <AppRoute
+          path='/admin/workspace-audit'
+          component={() => <WorkspaceAuditPage routeData={{title: 'Workspace Audit'}}/>}
+      />
+      <AppRoute
+          path='/admin/workspace-audit/:workspaceNamespace'
+          component={() => <WorkspaceAuditPage routeData={{title: 'Workspace Audit'}}/>}
+      />
+      <AppRoute
           path='/data-code-of-conduct'
           component={ () => <DataUserCodeOfConductPage routeData={{
             title: 'Data User Code of Conduct',
             minimizeChrome: true
           }} />}
-        />
-        <AppRoute path='/admin/user-audit'
-                  component={() => <UserAuditPage routeData={{title: 'User Audit'}}/>}/>
-        <AppRoute path='/admin/user-audit/:username'
-                  component={() => <UserAuditPage routeData={{title: 'User Audit'}}/>}/>
-        <AppRoute path='/admin/workspace-audit'
-                  component={() => <WorkspaceAuditPage routeData={{title: 'Workspace Audit'}}/>}/>
-        <AppRoute path='/admin/workspace-audit/:workspaceNamespace'
-                  component={() => <WorkspaceAuditPage routeData={{title: 'Workspace Audit'}}/>}/>
+      />
+      <AppRoute path='/nih-callback' component={() => <HomepagePage routeData={{title: 'Homepage'}}/>} />
+    </ProtectedRoutes>
+
+    <ProtectedRoutes guards={[signInGuard, registrationGuard]}>
```

Comment:
worth noting: it caused confusing bugs when I tried to nest it inside the other ProtectedRoutes. so we'll probably have to enumerate all combinations of guards. not a huge deal since we only have a few.